x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/i18n_texts.ts

```diff
@@ -189,27 +189,27 @@ export const i18nTexts = {
         defaultMessage: 'Cold phase',
       }),
       delete: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.deletePhase.deletePhaseTitle', {
-        defaultMessage: 'Delete Data',
+        defaultMessage: 'Delete data',
       }),
     },
     descriptions: {
       hot: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.hotPhase.hotPhaseDescription', {
         defaultMessage:
-          'You actively store and query data in the hot phase. All policies have a hot phase.',
+          'You store and query your data on hot nodes, your most performant hardware. Highest search speed for the biggest cost.',
       }),
       warm: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.warmPhase.warmPhaseDescription', {
         defaultMessage:
-          'You are still querying your index, but it is read-only. You can allocate shards to less performant hardware. For faster searches, you can reduce the number of shards and force merge segments.',
+          'Your data is read-only and you query it on warm nodes, less performant hardware. Slightly slower search for the smaller cost.',
```

Comment:
Data isn't strictly read-only in the warm phase. I like the imperative, but this sounds a bit counter-intuitive--whey would I want to move my frequently-searched indices to less performant hardware? Maybe:

```suggestion
          'Move data to the warm tier, which is optimized for search performance over indexing performance. Data is infrequently added or updated in the warm phase.',
```